src/platforms/apple/troubleshooting.mdx

```diff
@@ -35,3 +35,9 @@ If you use Tuist to manage your SPM dependencies, add the following configuratio
 ```Swift
 "Sentry": ["USE_HEADERMAP": "YES"]
 ```
+
+## SentryFileManager: Failed to write data to path
+
+Sentry needs to write files in a specific path in the app cache directory. This path is created during the SDK initialization.
+If you see a log that looks like this: `[SentryFileManager:439] Failed to write data to path` it means that the cache path don`t exist anymore.
+Check your app for any procedure that cleans the cache of your app during runtime, and avoid deleting Sentry directory.
```

Comment:
`h`: How to avoid this? Would need to know what the path is called and exclude that?

Reply:
```suggestion
Check your app for any procedure that cleans the cache of your app during runtime, and avoid deleting `io.sentry` directory.
```